kube.libsonnet

```diff
@@ -53,10 +53,15 @@
 // (client-side, and gives better line information).
 
 {
+  // In case you may want/need to skip assertions for speed reasons (rather big configmaps/etc),
+  // load the library with e.g.
+  //   local kube = (import "lib/kube.libsonnet") { _assert:: false };
+  _assert:: true,
+
   // resource contructors will use kinds/versions/fields compatible at least with version:
   minKubeVersion: {
     major: 1,
-    minor: 9,
+    minor: 14,
```

Comment:
ooc, why the minor release is `14`?

Is that the k8s version where Kubernetes started supporting the `apiVersion: apps/v1` and all the recent updates you made to this library?thx

Reply:
Exactly as you said -- the idea is to manifest (to library users) the minimum `<major>.<minor>` Kubernetes release required by the library.